docs/tutorials/blog.md

```diff
@@ -962,13 +967,15 @@ For some real fun, disable JavaScript in your dev tools and try it out. Because
 
 💿 Slow down our action with a fake delay
 
-```tsx filename=app/routes/admin/new.tsx lines=[5]
+```tsx filename=app/routes/admin/new.tsx lines=[5-6]
 // ...
 export const action: ActionFunction = async ({
   request
 }) => {
   await new Promise(res => setTimeout(res, 1000));
+
   const formData = await request.formData();
+
```

Comment:
This new line is part of the previous version of this function, so in order for people to orient themselves in an easier way I thought it should be reflected in this updated version as well.